src/time.cr

```diff
@@ -481,13 +466,18 @@ struct Time
   end
 
   def inspect(io : IO)
-    Format.new("%F %T").format(self, io)
-
     case
     when utc?
-      io << " UTC"
+      to_s "%F %T UTC", io
     when local?
-      Format.new(" %:z").format(self, io)
+      to_s "%F %T", io
```

Comment:
I don't like that the string representation of time depends on the local timezone of the computer. The string representation should be purely UTC or an offset. Omitting the offset when the offset happens to be the local timezone will probably lead to bugs.

Reply:
The offset is not omitted if it matches the local timezone but only if this instance has `location == Location.local` But as @asterite also preferes option D) I will update that.